src/agenteval/cli.py

```diff
@@ -140,6 +140,39 @@ def verify_git_reproducibility() -> None:
         )
 
 
+def check_results_against_eval_config(
+    results: TaskResults, eval_config: EvalConfig, eval_config_origin: str
+):
+    # warn about results missing
+    if len(results.task_names) == 0:
+        click.echo(f"Warning: no results.")
+
+    # warn about tasks missing from results
+    missing_tasks = eval_config.task_names - results.task_names
+    if len(missing_tasks) > 0:
```

Comment:
I don't think this strictly causes issues for the view part, but this seems like a sensible place for this to live, and seems like good information to have along side any missing primary metrics for tasks that are there (that's what comes next).

Reply:
I did! This PR moves that check into this helper (for both the score command and the lb publish command)